lib/app_scaffold/dist/assets/iframe.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <link href="app.css" rel="stylesheet">
+  </head>
+
+  <body>
+    <section data-main class="main">
+      <img class="loader" src="dot.gif"/>
+    </section>
+
+    <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+    <script type="text/javascript" src="app.js"></script>
+</body>
```

Comment:
👉

Reply:
! I was like,
> What does this mean?

It's about indentation right?

Reply:
Oh yes sorry. The indentation doesn't look aligned here.